frontend/src/assets/css/styles.scss

```diff
@@ -0,0 +1,39 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+    html {
+        font-family: Manrope, sans-serif;
```

Comment:
Wouldn't be sufficient to add the font-sans class to the body in index.html file since you've already redefined it in the config?